org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterIas.java

```diff
@@ -109,6 +103,21 @@ protected boolean supportsIasChannel(ZigBeeEndpoint endpoint, ZoneTypeEnum requi
         return zoneType == requiredZoneType;
     }
 
+    protected boolean hasIasZoneInputCluster(ZigBeeEndpoint endpoint) {
+        if (endpoint.getInputCluster(ZclIasZoneCluster.CLUSTER_ID) == null) {
```

Comment:
I don't quite see the difference between the two `if` statements in this method - don't they just test the same? Or do I overlook something here?

(I know that this code was copied from the battery converter - but nevertheless think that this is a valid question :) )

Reply:
I took a closer look to it and I agree that both statements do the same.
I can delete the second statement and then have this method body
``` 
        if (endpoint.getInputCluster(ZclIasZoneCluster.CLUSTER_ID) == null) {
            logger.trace("{}: IAS zone cluster not found", endpoint.getIeeeAddress());
            return false;
        }

        return true;
```

Reply:
Sounds good :+1: